pkgs/servers/flaresolverr/default.nix

```
@@ -0,0 +1,52 @@
+{ pkgs, lib, stdenv, python3 }:
+with pkgs;
+let
+  python_env = pkgs.python3.withPackages
+    (p: with p; [ bottle waitress selenium func-timeout requests websockets xvfbwrapper webtest ]);
+in
+stdenv.mkDerivation
+rec {
+
+  pname = "flaresolverr";
+  version = "3.0.2";
+
+  src = fetchFromGitHub {
+    owner = pname;
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-zpeJf1CaQ4bsncZz44sH+tFKddYrZf7YdNYL50d9GA4=";
+  };
+
+  buildInputs = [ pkgs.makeWrapper ];
+  nativeBuildInputs = [ chromedriver ];
+  patches = [ ./chromedriver_path.patch ];
+
+  postPatch = ''
+    substituteInPlace src/utils.py \
+    --replace "NIX_STORE_DRIVER_PATH" "${pkgs.chromedriver}/bin/chromedriver"
+  '';
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/share
+    cp -r src $out/share
+    cp package.json $out/share
+    runHook postInstall
+  '';
+
+  postFixup = ''
+    makeWrapper ${python_env}/bin/python $out/bin/flaresolverr \
+      --prefix PATH : ${lib.makeBinPath [ pkgs.chromium pkgs.xvfb-run xorg.xorgserver pkgs.chromedriver ]} \
```

```suggestion
      --prefix PATH : ${lib.makeBinPath [ chromium xvfb-run xorgserver chromedriver ]} \
```
Why does it need xorgserver?